R/xml-column-names-aglu.R

```diff
@@ -1,7 +1,17 @@
+# XML-related constants
+
+aglu.GLU_NDIGITS          <- 3    # number of digits in the geographic land unit identifier codes
+aglu.GLU_NAME_DELIMITER   <- ""   # delimiter between the GLU name and number
+aglu.LT_GLU_DELIMITER     <-      # delimiter between the land use type name and GLU name. should be the same as the crop-glu delimiter
+  aglu.CROP_GLU_DELIMITER <- "_"  # delimiter between the crop name and GLU name
```

Comment:
Notice the `aglu.LT_GLU_DELIMITER` is assign to it in previous line (i.e. all one line syntactically).